src/com/chocohead/eumj/EngineMod.java

```diff
@@ -47,7 +47,7 @@
 import com.chocohead.eumj.te.Engine_TEs;
 import com.chocohead.eumj.te.TileEntityEngine;
 
-@Mod(modid=MODID, name="EU-MJ Engine", dependencies="required-after:ic2;required-after:buildcraftenergy@[7.99.16,7.99.17];after:buildcrafttransport", version="@VERSION@")
+@Mod(modid=MODID, name="EU-MJ Engine", dependencies="required-after:ic2;required-after:buildcraftenergy@[7.99.21];after:buildcrafttransport", version="@VERSION@")
```

Comment:
I'm not sure if this is actually a sensible version range or not :/

Reply:
The original idea was that each BuildCraft release was breaking something so it would need recompiling anyway to avoid crashing, thus the restrictive version ranges. This was also under the impression I'd keep up with updates as BC did, so maybe it can be relaxed and the incompatibilities fixed as they come instead...